int8 num #num of servo
int8 channel #канал на плате
int8 speed #in %
int16 min_val #В тугриках
int16 max_val #В тугриках
---
string resp